src/net.h

```diff
@@ -677,6 +677,7 @@ class CNode
 
 
     uint64_t nLocalHostNonce;
+    // What local services you advertised on connection to this node
```

Comment:
Yes, the intention is to be able to (in the future) offer different services to different peers. For example, incoming vs outgoing, proxy, tor, whitelist, etc. Even if we don't do that now, it's helpful to think in terms of "what did I tell this peer I'm offering?" rather than a global value. For now each connman instance (only 1 for now) passes along the same services to all nodes. 

ACK on clarifying, though I'd say something like "Services offered to this peer".